### PR TITLE
Cherry Pick 1408 into release 0.26 - Enable chart deploy on github workflow (#1408)

### PR DESCRIPTION
**Pull request by AlfredoG87**
**Description**:
Cherry-pick PR #1408 into release/026

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,7 +192,7 @@ helm show values hedera-json-rpc-relay
 Deploy an installation with custom values file:
 
 ```
-helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./helm-chart --debug
+helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./charts --debug
```
Comment (Nana-EC) [inline README.md:195]:
```suggestion
helm install custom-hedera-json-rpc-relay -f path/to/values/file.yaml ./charts/hedera-json-rpc --debug
```

Comment (Nana-EC) [reply README.md:195]: I made this mistake but will fix it in main

**Review by Nana-EC — APPROVED** [review]
LG, if you can fix my omission as a nit that would be great

File: README.md (hunk 1 above)
Comment (AlfredoG87) [reply README.md:195]: yeah, I will update thos PR to include that.

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**